tools/sass_bundle.ts

```diff
@@ -0,0 +1,29 @@
+import {dirname, join} from 'path';
```

Comment:
don't you need license headers in this repo?

Reply:
As far as I knew we only needed license headers for code synced into google.